### PR TITLE
EPMDJ-6736: implement expand/collapse component

### PR DESCRIPTION
**Pull request by IliaSolovev**
Issue: EPMDJ-6736

### CODE REVIEW

File: src/css/_expand.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+$box-shadow: 0px 0px 12px rgba(88, 98, 109, 0.07), 0px 2px 4px rgba(88, 98, 109, 0.3);
```
Comment (IliaSolovev) [inline src/css/_expand.scss]: I think that it should be in another file but I don`t how give a name for it)

File: src/css/_expand.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  &:hover{
+    outline: none;
+    text-decoration: none;
+    color: map-get($monochrome, default);
+    box-shadow: $box-shadow-on-hover;
+    transform: scale(1.01);
+  }
+}
+
+details summary::after{
```
Comment (RomanDavlyatshin) [inline src/css/_expand.scss]: Consider nesting that in the `details summary {` rule above

File: src/css/_expand.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  content: url(/static/img/arrow.svg);
```
Comment (RomanDavlyatshin) [inline src/css/_expand.scss]: I believe that the value supplied to `url()` must be quoted.

File: src/css/_expand.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  position: absolute;
+  right: 28px;
+  z-index: 100;
+}
+
+details[open] summary::after {
```
Comment (RomanDavlyatshin) [inline src/css/_expand.scss]: Consider nesting that as well.
Just implement breakdown a `details summary {` rule above into two levels.

File: src/css/_expand.scss (hunk 1 above)
Comment (RomanDavlyatshin) [reply src/css/_expand.scss]: If these shadows repeat across the project, create mixins and place them in separate `mixins.scss` file.

File: src/css/_expand.scss [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  padding: 24px;
+  font-family: "OpenSans-Semibold", sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  color: map-get($monochrome, default);
+  background-color: map-get($monochrome, white);
+  box-shadow: $box-shadow;
+  border-radius: 8px;
+  cursor: pointer;
+  transition: box-shadow 200ms ease, transform 200ms ease;
```
Comment (RomanDavlyatshin) [inline src/css/_expand.scss]: This transition should be a part of a `shadow` mixin as well

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]

File: src/css/_mixins.scss [hunk 6]
```diff
@@ -0,0 +1,9 @@
+@mixin card-shadow {
```
Comment (IliaSolovev) [inline src/css/_mixins.scss:1]: I gave so name because so shadow use only in the block that looks like a card

**Review by RomanAchilov — APPROVED** [review]

**Comment by RomanDavlyatshin** [comment]
outdated

**Pull request closed without merging**